srcpkgs/esptool/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'esptool'
+pkgname=esptool
+version=3.1
+revision=1
+wrksrc="esptool-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip"
+depends="python3-pyserial python3-ecdsa python3-pyaes python3-bitstring python3-reedsolo"
```

Comment:
Break at ~80 columns (I'm on phone, I think this exceeds that).

Reply:
Keen eye sir. This could be a feature for `xlint` :)